compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/repositories/RemotePackageRepository.java

```diff
@@ -101,12 +101,14 @@ public static RemotePackageRepository from(Environment environment, Path cacheDi
         String orgName = resolutionRequest.orgName().value();
         String packageName = resolutionRequest.packageName().value();
 
-        //If environment is offline we return the local versions
+        // First, Get local versions
+        List<PackageVersion> packageVersions = new ArrayList<>(fileSystemRepo.getPackageVersions(resolutionRequest));
```

Comment:
Since this is a list, can we get duplicate versions?

Reply:
I think yes, Will fix soon